api/wasm/cpp/proxy_wasm_intrinsics.pb.h

```diff
@@ -1,8 +1,8 @@
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: proxy_wasm_intrinsics.proto
+// source: api/wasm/cpp/proxy_wasm_intrinsics.proto
```

Comment:
How did you generate this? Somehow, the path and object names changed to include the directory, compared to the version that's currently checked in (though, I'm not sure if this matters at all).

Reply:
I ran `protoc --cpp_out=./ -I./ -Iapi/wasm/cpp  api/wasm/cpp/proxy_wasm_intrinsics.proto` under root directory to generate the code. I think it does not matter, when building with null, the plugin uses GrpcService from upstream. The generated code is used in wasm module, where include path depends. But I think it is better to keep it in sync with lite version, so I reran the command within `api/wasm/cpp` and now those path diff should be gone.